src/main/java/uk/gov/ons/census/action/model/entity/ActionRule.java

```diff
@@ -34,7 +27,6 @@
 
   @Column private Boolean hasTriggered;
 
-  @Type(type = "jsonb")
-  @Column(columnDefinition = "jsonb", nullable = false)
-  private Map<String, List<String>> classifiers;
+  @Column(nullable = false, length = 100000)
+  private String userDefinedWhereClause;
```

Comment:
it's not really 'user defined'. I think `classifierClause` is probably more descriptive, personally.

Reply:
or just `classifiers` because its functionality hasn't changed at all